UIforETW/ChildProcess.cpp

```diff
@@ -112,10 +111,10 @@ DWORD ChildProcess::ListenerThread()
 	return 0;
 }
 
-
+_Pre_satisfies_(!(this->hProcess_))
 bool ChildProcess::Run(bool showCommand, std::wstring args)
 {
-	assert(!hProcess_);
+	ATLASSERT(!hProcess_);

```

Comment:
Lets take it as a given that ATLASSERT is enough better than assert to justify the churn of changing all instances. Even with assumption, is ATLASSERT perfect? Probably not. So why not change ATLASSERT to UIETWASSERT or some-such, and #define that as ATLASSERT. Then if we ever want to change it back, or to an assert that is actually good, it won't require any churn.

Just a thought. Not guaranteed to be a good on.